packages/common-assets/gulpfile.js

```diff
@@ -0,0 +1,38 @@
+const gulp = require("gulp");
+const gulpless = require("gulp-less");
+const postcss = require("gulp-postcss");
+const debug = require("gulp-debug");
+const csso = require("gulp-csso");
+const autoprefixer = require("autoprefixer");
+const NpmImportPlugin = require("less-plugin-npm-import");
+
+gulp.task("less", () => {
+  const plugins = [autoprefixer()];
+
+  return gulp
+    .src("assets/themes/*-theme.less")
+    .pipe(debug({ title: "Less files:" }))
+    .pipe(
+      gulpless({
+        javascriptEnabled: true,
+        plugins: [new NpmImportPlugin({ prefix: "~" })],
+      })
+    )
+    .pipe(postcss(plugins))
+    .pipe(
+      csso({
+        debug: true,
+      })
+    )
+    .pipe(gulp.dest("./public"));
```

Comment:
the assets in `public` should be added to `.gitignore` in `common-assets` since there are derived from other files

Reply:
Removing public causes [build failure](https://github.com/dendronhq/dendron/runs/6720635396?check_suite_focus=true#step:10:77)

Reply:
the reason you are getting this error is because `dendron-plugin-views` used to copy from `dendron-next-server/public`. we've moved the logic that generates the `public` folder from `dendron-next-server` to `common-assets` but we never invoke it. we need to update the `build` step of `common-assets`

```
    "build": "node scripts/buildStyles.js && yarn setup",
```